src/app/not-found.tsx

```diff
@@ -0,0 +1,67 @@
+"use client";
+
+import { CircleNotch, Plugs } from "phosphor-react";
+import { useState } from "react";
+import { useRouter } from "next/navigation";
+import { toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
+import { Button } from "@/components";
+
+export default function NotFound() {
+  const router = useRouter();
+  const [isRedirecting, setIsRedirecting] = useState<boolean>(false);
+
+  async function handleRedirectUser(): Promise<void> {
+    setIsRedirecting(true);
+    try {
+      router.push("/");
+      await new Promise((resolve) => setTimeout(resolve, 3000));
```

Comment:
Qual a necessidade do timeout?

Reply:
Era pra forçar uma atraso e ver o spin, esqueci de tirar